rust/Earthfile

```diff
@@ -1,30 +1,37 @@
-VERSION 0.7
+VERSION --global-cache 0.7
 
 # CARGO runs the cargo command "cargo $args".
 # This UDC should be thread safe. Parallel builds of targets using it should be free of race conditions.
 # Arguments:
 #   - args: Cargo subcommand and its arguments. Required
 #   - keep_fingerprints (false): Do not remove source packages fingerprints. Use only when source packages have been COPYed with --keep-ts option
+#   - sweep_days (4): Number of days to retain target build artifacts in cache
 #   - output: Regex to match the files within the target folder to be copied from the cache to the caller filesystem (image layers).
 #     Use this argument when you want to SAVE an ARTIFACT from the target folder (mounted cache), always trying to minimize the total size of the copied fileset.
 #     For example --output="release/[^\./]+" would keep all the files in /target/release that don't have any extension
 CARGO:
     COMMAND
     ARG --required args
     ARG keep_fingerprints=false
+    ARG sweep_days=4
     ARG output
```

Comment:
I think `args`, `sweep_days` and `output` can be declared further down after the call to `INSTALL_CARGO_SWEEP` so it wouldn't bust the cache for REMOVE_SOURCE_FINGERPRINTS

Reply:
These args are unlikely to change and +REMOVE_SOURCE_FINGERPRINTS runs in a few seconds. 
Under these circumstances. I prefer leaving it as it is since I find more recreable having the ARGS in the beginning.
Having arg values affect the cache seems broken to me, topic for another discussion :)